src/main/java/io/cdap/plugin/dlp/DLPTransformPluginConfig.java

```diff
@@ -186,4 +196,19 @@ public void validate(FailureCollector collector, Schema inputSchema) {
       }
     }
   }
+
+  public String getDlpLocation() {
+    return dlpLocation;
+  }
+
+  public String getDlpParentResourceLocationUtil(String project, String location) {
```

Comment:
Can we return valid `LocationName` object instead of raw string? We can convert it to string whenever needed using `toString()`.

Reply:
done.